kod/object/active/flag.kod

```diff
@@ -709,7 +708,11 @@ messages:
          }
       }
 
+      % If setting the faction as part of a bulk process (i.e. recreate),
+      % pass `claim=FALSE` and be sure to manually recalculate the flags 
+      % and save the flag states, as `FlagClaimed` does.
```

Comment:
Comments like "as FlagClaimed does" typically "rot", as they're referring to code that can be changed independently.  10 years from now, someone will look for "FlagClaimed", and maybe it will be gone.